File: Br.Com.Fiap.Gere.Residuo/Model/CaminhaoModel.cs
namespace Br.Com.Fiap.Gere.Residuo.Model
{
    public class CaminhaoModel
    {
        public int CaminhaoId { get; set; }

        // Relacionamento com AgendaModel
        public List<AgendaModel> AgendasCriadasComEsteCaminhao { get; set; }

        public string CaminhaoPlaca { get; set; }
        public DateOnly DataFabricacao { get; set; }

        public string CaminhaoMarca { get; set; }

        public string CaminhaoModelo { get; set; }

        public bool CaminhaoEstaDisponivel { get; set; } = true;


    }
}

File: BDD_Tests/BDD-Gere-Residuo/src/test/java/model/caminhao/CaminhaoModel.java
package model.caminhao;

import com.google.gson.annotations.Expose;
import lombok.Data;

import java.util.Date;

@Data
public class CaminhaoModel {

    @Expose(serialize = false)
    private int caminhaoId;
    @Expose(serialize = false)
    private boolean caminhaoEstaDisponivel;
    @Expose
    private String caminhaoPlaca;
    @Expose
    private String dataFabricacao;
    @Expose
    private String caminhaoMarca;
    @Expose
    private String caminhaoModelo;
}
